Jenkinsfile

```diff
@@ -661,6 +665,65 @@ pipeline {
       }
     }
 
+    stage('Deploy Preview') {
+      when {
+        not {
+          branch 'PR-*'
+        }
+      }
+      steps {
+        setGitHubBuildStatus('nuxeo/preview/deploy', 'Deploy nuxeo Preview', 'PENDING')
+        container('maven') {
+          dir('ci/helm/preview') {
+            echo """
+            ----------------------------------------
+            Deploy Preview environment
+            ----------------------------------------"""
+            // first substitute docker image names and versions
+            sh """
+              mv values.yaml values.yaml.tosubst
+              envsubst < values.yaml.tosubst > values.yaml
+            """
+            // second create target namespace (if doesn't exist) and copy secrets to target namespace
+            script {
+              sh(returnStdout: true, script: 'jx -b ns | sed -r "s/^Using namespace \'([^\']+)\'.+\\$/\\1/"')
```

Comment:
Is this line really useful?

Reply:
doesn't work anyway, this is what is litterally printed actually by jx:
```
sh(returnStdout: true, script: 'jx -b ns | sed -r "s/^Using namespace \'([^\']+)\'.+\\$/\\1/"')
```

Reply:
Maybe because the result of the shell script isn't assigned to any variable.

Just for information, if it were assigned to a variable, it would return the name of the current namespace, you can test it locally without the escaping characters:
```
$ jx -b ns | sed -r "s/^Using namespace '([^\']+)'.+\$/\\1/"
platform
```

Reply:
indeed `platform` in my case
I should have `echo`ed it